<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPerformance</class>
 <widget class="QDialog" name="AddPerformance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="mainLabel">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ДОБАВЛЕНИЕ/РЕДАКТИРОВАНИЕ СПЕКТАКЛЯ</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1" rowspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item row="11" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="buttonOK">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCANCEL">
         <property name="text">
          <string>CANCEL</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="namePerformance">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="producer"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="producerLabel">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ПОСТАНОВЩИК</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="namePerformanceLabel">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>НАЗВАНИЕ СПЕКТАКЛЯ</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="authorLabel">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>АВТОР ПЬЕСЫ</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="author"/>
     </item>
     <item row="9" column="0" colspan="3">
      <widget class="QLabel" name="labelError">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { color: rgb(240, 0, 0); }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="castLabel">
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>СПИСОК ДЕЙСТВУЮЩИХ ЛИЦ</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTableWidget" name="tableActors">
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>200</number>
       </attribute>
       <column>
        <property name="text">
         <string>Фамилия </string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Имя</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="10" column="1">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCANCEL</sender>
   <signal>clicked()</signal>
   <receiver>AddPerformance</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOK</sender>
   <signal>clicked()</signal>
   <receiver>AddPerformance</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
